doc/nmstatectl.8.in

```diff
@@ -0,0 +1,99 @@
+.\" Manpage for nmstatectl.
+.TH nmstatectl 8 "@DATE@" "@VERSION@" "nmstatectl man page"
+.SH NAME
+nmstatectl \- A nmstate command line tool
+.SH SYNOPSIS
+.B nmstatectl show \fR[\fIINTERFACE_NAME\fR] [\fB--json\fR]
```

Comment:
the `nmstatectl show` should also be rounded with `[\fI` and `\fR`.

And the leading `\fR` seems not required.

The same also apply to the following lines.

Reply:
Looking at other manpages, they all follow that pattern. Shouldn't we do it the same way?

Examples:
nmcli(1), NetworkManager(8), iptables(8)